### DISCUSSION TITLE
"No logbook found", but I have a logbook!

### DISCUSSION THREAD

**Discussion by emilyaviva** [magicbug/Cloudlog#2304] [Q&A] [answered]
Hey there, I'm trying to setup and host a local instance of Cloudlog, and I have created a station logbook, but for some reason it doesn't show up on the Dashboard, and just directs me to create one. Am I missing something incredibly basic, or is something not set up right, or what?

Fig. 1: error on dashboard page.
<img width="1275" alt="Screenshot showing the error on dashboard page." src="https://github.com/magicbug/Cloudlog/assets/69606468/db75c0fd-1aab-4693-8641-85fa04c0db4e">

Fig. 2: I do indeed have a logbook.
<img width="1275" alt="Screenshot showing I do indeed have a logbook." src="https://github.com/magicbug/Cloudlog/assets/69606468/89a064b9-21cb-433c-be08-d0fa92c8ea79">

Fig. 3: The logbook does have 2 QSOs in it, and is linked to a location:
<img width="1270" alt="Screenshot showing the logbook has 2 QSOs and is linked to a location." src="https://github.com/magicbug/Cloudlog/assets/12450261/e62a6cdd-15df-413c-8eb8-424c290b2d87">

Thanks!

**Comment by magicbug** [comment, marked as answer]
Even though you have a logbook you have to manually assign it some station locations to do this

Go back to Station Logbooks and edit the one you want to use then link the locations.

**Reply by emilyaviva** [reply]
Ah ha, you have to press the "link location" button! Thanks! 👍
